README.md

```diff
@@ -20,6 +20,22 @@ You can then run `helm search repo jenkins` to see the charts.
 <!-- Keep full URL links to repo files because this README syncs from main to gh-pages.  -->
 Chart documentation is available in [jenkins directory](https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/README.md).
 
+## Development
+
+### Prerequisites
+
+- [minikube](https://minikube.sigs.k8s.io/docs/start/), [k3d](https://k3d.io/), [kind](https://kind.sigs.k8s.io/), [microk8s](https://microk8s.io/) or some other local k8s cluster
+- [Tilt](https://tilt.dev/)
+
+### Setup
+
+```console
+tilt up
+# Open a service tunnel into the cluster with minikube
+minikube service chart-jenkins
+# Login with username admin and password asdf
+```
+
```

Comment:
I suggest to move this to CONTRIBUTING.md

Reply:
Done